geomstats/geometry/invariant_metric.py

```diff
@@ -1086,12 +1196,12 @@ def inner_product(self, tangent_vec_a, tangent_vec_b, base_point=None):
         return inner_prod
 
     def parallel_transport(self, tangent_vec_a, tangent_vec_b, base_point):
-        r"""Compute the parallel transport of a tangent vector.
+        r"""Compute the parallel transport of a tangent vec along a geodesic.
 
         Closed-form solution for the parallel transport of a tangent vector a
         along the geodesic defined by :math: `t \mapsto exp_(base_point)(t*
-        tangent_vec_b)`. As the special Euclidean group endowed with its
-        canonical left-invariant metric is a symmetric space, parallel
+        tangent_vec_b)`. As a Lie group endowed with its
+        canonical bi-invariant metric is a symmetric space, parallel
```

Comment:
Not all Lie groups have a bi-invariant metric: what do you mean here?

Reply:
As we are in the bi-invariant class, I was assuming the group considered has a bi-invariant metric! But I can add *compact* Lie group